### DISCUSSION TITLE
my qtile ram ram usage

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3048] [Q&A]
https://github.com/ArchKubi/MyArchFormatTest/tree/main/desktop_manager/low_resolution/qtile
i use arch linux default config file normally it uses 100 - 200 mb ram but my config file is 430-500 mb ram is this normal?

**Comment by m-col** [comment]
To track down the cause, introduce parts of your own config on top of the default config and see how they affect memory usage. Hopefully you'll find that a specific part causes most of the increase, and then it can be adjusted/fixed.

**Reply by ghost** [reply]
okay ı try
